nautobot/dcim/tests/test_views.py

```diff
@@ -769,6 +769,7 @@ def setUpTestData(cls):
         Platform.objects.all().delete()
         Module.objects.all().delete()
         ModuleType.objects.all().delete()
+        CloudService.objects.all().delete()
         CloudNetwork.objects.all().delete()
         CloudAccount.objects.all().delete()
         CloudType.objects.all().delete()
```

Comment:
Can we implement `get_deletable_object` and `get_deletable_object_pks` on this test case (as per the TODO comment above) so that we don't have to keep updating it like this?